### ISSUE TITLE
port latest neoforge 1.21.1 update to cf

### ISSUE THREAD

**Issue by VaporeonScripts** [PieKing1215/InvMove#70]

**Comment by PieKing1215** [comment]
Latest release is [available on CF](https://www.curseforge.com/minecraft/mc-mods/invmove/files/all?page=1&pageSize=22&showAlphaFiles=show)!

**Comment by VaporeonScripts** [comment]
yippie, will you also do the same with invmove compats?

**Comment by PieKing1215** [comment]
Yeah next update for compats will do the same, I won't be able to get it out for at least a week though
